### PR TITLE
Add endpoint to apply modifications on a network, using a list of existing modifications uuids.

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/NetworkModificationController.java [hunk 1]
```diff
@@ -269,4 +269,15 @@ public ResponseEntity<Void> deleteStashedModificationInGroup(@Parameter(descript
     public ResponseEntity<List<ModificationMetadata>> getModificationsMetadata(@RequestParam("ids") List<UUID> ids) {
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(networkModificationService.getModificationsMetadata(ids));
     }
+
+    @PutMapping(value = "/network-modifications/uuids", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/NetworkModificationController.java:273]: it's weird to have "uuids" here ...

Why not "/network-modifications/apply" ?

To be confirmed with Slimane ?

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: a request/remark about REST naming
tests: ok

File: src/main/java/org/gridsuite/modification/server/NetworkModificationController.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/NetworkModificationController.java]: Renamed into : "/networks/{networkUuid}/apply"

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Pull request merged**